pkg/spec/spec.go

```diff
@@ -32,7 +32,8 @@ type Function struct {
 // FunctionSpec contains func specification
 type FunctionSpec struct {
 	Handler  string             `json:"handler"`
-	Function string             `json:"function"`
+	File     string             `json:"file"`
```

Comment:
Should document (comment) what values are expected here, since `string` doesn't give much of a hint.